IdSSLOpenSSLHeaders.pas

```diff
@@ -15674,26 +15771,30 @@   STACK_OF_SRTP_PROTECTION_PROFILE = record
   {$EXTERNALSYM PSTACK_OF_SRTP_PROTECTION_PROFILE}
   PSTACK_OF_SRTP_PROTECTION_PROFILE = PSTACK;
   {$ENDIF}
+
+
   {$NODEFINE SSL_CTX}
   SSL_CTX = record
+    libctx: Pointer;
     method: PSSL_METHOD;
     cipher_list: PSTACK_OF_SSL_CIPHER;

```

Comment:
We aren't meant to access the SSL_CTX structure fields directly. You can use the SSL_CTX_get_ex_data and SSL_CTX_set_ex_data to pass callback data for the IndySSL_CTX_use_PrivateKey_file_PKCS12, IndySSL_CTX_use_certificate_file_PKCS12, IndySSL_CTX_use_PrivateKey_file, IndySSL_CTX_use_certificate_file, and IndySSL_CTX_use_certificate_chain_file.